hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStore.java

```diff
@@ -151,20 +159,59 @@ public synchronized void open(HugeConfig config) {
             return;
         }
 
+        List<Future<?>> futures = new ArrayList<>();
+        ExecutorService openPool = ExecutorUtil.newFixedThreadPool(8, DB_OPEN);
```

Comment:
can also use MAX_FILE_OPENING_THREADS?

Reply:
this thread pool is used to open DBs, not RocksDB sst files.